### ISSUE TITLE
Changing Quality Resets Actual Speed Without Updating Menu

### ISSUE THREAD

**Issue by DouglasGreen** [sampotts/plyr#1436]
### Expected behaviour

Changing quality does not affect speed.

### Actual behaviour

Changing quality resets the actual speed to normal without updating the menu.

### Steps to reproduce

This issue is present in my locally installed copy of plyr as well as in the demo version. You can reproduce the issue in the demo version, so it's not just me.

1. Go to https://plyr.io/.
2. Play the video and make sure volume is audible so you can hear the difference.
3. Open the Settings menu and change the Speed to something noticeably different (fast or slow).
4. Open the Quality menu and change quality.
5. See how the actual Speed resets to Normal without the menu setting being updated.

I think what needs to happen is that when video quality is changed and the other video starts playing, the speed needs to be set again from its stored value.

### Environment

- Browser: Chrome or Firefox
- Version: latest
- Operating System: Linux or Windows
- Version: latest

### Console errors (if any)

None

### Link to where the bug is happening

https://plyr.io/

**Comment by indrajaala** [comment]
yes, whenever source changes speed option is restored to default, not being able to set speed in a setup where source changes dynamically.
